mtgbp/assetlist.json

```diff
@@ -0,0 +1,46 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "mtgbp",
+    "assets": [
+      {
+        "description": "The native staking and governance token of mtgbp.",
+        "extended_description": "mtgbp is a Layer 1 protocol that allows developers to automatically spin up VM-agnostic, parallelized and interoperable dedicated chains, or “Chainlets,” that provide applications with infinite horizontal scalability. Each Chainlet is a replica of the mtgbp Mainnet, with the same validator set and security model.\n\nmtgbp’s mission is to enable the next 1000 chains in gaming and entertainment as part of the growing mtgbp Multiverse.",
+        "denom_units": [
+          {
+            "denom": "mtgbp",
+            "exponent": 0
+          }
+        ],
+        "base": "mtgbp",
+        "name": "mtgbp",
+        "display": "mtgbp",
+        "symbol": "MTGBP",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/mtgbp/images/mtgbp.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/mtgbp/images/mtgbp.svg"
+        },
+        "coingecko_id": "ttnc",
```

Comment:
This CGID doesn't exist at https://api.coingecko.com/api/v3/coins/list. Please remove